start.js

```diff
@@ -0,0 +1,26 @@
+const {server} = require('./app.js');
+const { getPool } = require('./db');
+
+const pool = getPool();
+
+const PORT = process.env.PORT || 5000;
+const HOST = process.env.HOST || '127.0.0.1';
+
+server.listen(PORT, HOST, () => {
+    console.log(`Server is listening on http://${HOST}:${PORT}`);
+    pool.connect()
+    .then(() => {
+        console.log("Connected to DB");
+    })
+    .catch((error) => {
+        console.error(error);
+    })
+})
+
+server.on("close", () => {
+    console.log("Closing connection to DB");
+    pool.end() // TODO: return pool.end()? 
```

Comment:
Can't return from inside the event listener - why were you planning to do that?

Reply:
Line 120 in `seed.js` of DNDB had `return client.end()`, and I didn't know if there was some functionality with the pg api that I didn't understand because I hadn't used a return when I did it. It was a note to check in about it. Was that return on line 120 a typo? It was was in a different context (in a `.then()` chain instead of an event listener).

Reply:
Yes, that was different because of the .then

When in a .then chain, each callback passed to .then can return a promise. That promise, once resolved, will be passed as the argument to the callback in the next .then in the chain